UIKitDZ/UIKitDZ/Modules/OrderModule/Controller/OrderViewController.swift

```diff
@@ -0,0 +1,137 @@
+// OrderViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Order details screen
+class OrderViewController: UIViewController {
+    // MARK: - Private Properties
+
+    private lazy var dismissButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: "dismissButton"), for: .normal)
+        button.addTarget(
+            self,
+            action: #selector(didTapDismissButton(_:)),
+            for: .touchUpInside
+        )
+        return button
+    }()
+
+    private var leftImageView: UIImageView = {
+        let view = UIImageView()
+        view.image = UIImage(named: "leftImage")
+        return view
+    }()
+
+    private var rightImageView: UIImageView = {
+        let view = UIImageView()
+        view.image = UIImage(named: "rightImage")
+        return view
+    }()
+
+    private var orderLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = UIColor(hex: "#111111")
+        label.text = "Вашъ Заказъ"
+        label.textAlignment = .center
+        label.font = .systemFont(ofSize: 18, weight: .bold)
+        return label
+    }()
+
+    private lazy var americanoLabel = createLabelWithText(
+        "Американо", alignement: .left
+    )
+    private lazy var milkLabel = createLabelWithText(
+        "Молоко", alignement: .left
+    )
+    private lazy var espressoLabel = createLabelWithText(
+        "Эспрессо 50мл", alignement: .left
+    )
+    private lazy var americanoPriceLabel = createLabelWithText(
+        "100 руб", alignement: .right
+    )
+    private lazy var milkPriceLabel = createLabelWithText(
+        "50 руб", alignement: .right
+    )
+    private lazy var espressoPriceLabel = createLabelWithText(
+        "50 руб", alignement: .right
+    )
+
+    private var orderPriceLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = UIColor(hex: "#111111")
+        label.text = "Цѣна - 200 руб"
+        label.textAlignment = .center
+        label.font = .systemFont(ofSize: 18, weight: .bold)
+        return label
+    }()
+
+    private var middleImageView: UIImageView = {
+        let view = UIImageView()
+        view.image = UIImage(named: "middleImage")
+        return view
+    }()
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        setupSubViews()
+        configureSubviews()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        let price = CoffeeViewController.currentPrice
```

Comment:
```suggestion
        let price = CoffeeViewController.currentPrice
        orderPriceLabel.text = "Цѣна - \(price) руб"
```
в жизненом цикле вызываем только методы